src/OrleansAWSUtils/Streams/SQSBatchContainer.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Amazon.SQS.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Orleans.Providers.Streams.Common;
+using Orleans.Runtime;
+using Orleans.Serialization;
+using Orleans.Streams;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using SQSMessage = Amazon.SQS.Model.Message;
+
+namespace OrleansAWSUtils.Streams
+{
+    [Serializable]
+    internal class SQSBatchContainer : IBatchContainer
```

Comment:
All this stream implementation is almost the same as the Azure. In fact, almost all Queue-based implementation will be the same...

I would yes think on a refactory on both Azure and AWS Utils and move common code on a base library but, I think there are too much things in state today to mess with the production code now of Azure. But yes, I agree with you, will make a note here and have a look at it later on.